src/_P039_Thermosensors.ino

```diff
@@ -405,9 +423,21 @@ boolean Plugin_039(byte function, struct EventStruct *event, String& string)
           }
           {
             addFormNote(F("Set Thermocouple type for MAX31856"));
-            const String Toptions[8]      = { F("B"), F("E"), F("J"), F("K"), F("N"), F("R"), F("S"), F("T") };
-            const int    ToptionValues[8] = { 0, 1, 2, 3, 4, 5, 6, 7 };
-            addFormSelector(F("Thermocouple type"), F("P039_tctype"), 8, Toptions, ToptionValues, P039_TC_TYPE);
+            const String Toptions[10]      = { F("B"), F("E"), F("J"), F("K"), F("N"), F("R"), F("S"), F("T"), F("VM16"), F("VM32") };
+            const int    ToptionValues[10] = { 0, 1, 2, 3, 4, 5, 6, 7, 8, 12};
+            addFormSelector(F("Thermocouple type"), F("P039_tctype"), 10, Toptions, ToptionValues, P039_TC_TYPE);
+          }
+          {
+            const String Coptions[5] = {F("1 sample"), F("2 samples"), F("4 samples"), F("8 samples"), F("16 samples")};
```

Comment:
Is the word "sample" needed here? Why not add it as a unit next to the selector (`addUnit`)

Reply:
Sure - saves some ROM and redundancy without loss of clarity. I am with you !

Reply:
Solved in next push.